resources/eventing/charts/controller/templates/deployment.yaml

```diff
@@ -58,18 +39,41 @@ spec:
                 key: client_secret
           - name: WEBHOOK_TOKEN_ENDPOINT
             value: https://oauth2.{{ .Values.global.domainName }}/oauth2/token
-          - name: DOMAIN
-            value: {{ .Values.global.domainName }}
-          - name: EVENT_TYPE_PREFIX
-            value: {{ .Values.global.eventTypePrefix }}
+          - name: PUBLISHER_REQUESTS_CPU
+            value: {{ .Values.publisherProxy.resources.requests.cpu }}
+          - name: PUBLISHER_REQUESTS_MEMORY
+            value: {{ .Values.publisherProxy.resources.requests.memory }}
+          - name: PUBLISHER_LIMITS_CPU
+            value: {{ .Values.publisherProxy.resources.limits.cpu }}
+          - name: PUBLISHER_LIMITS_MEMORY
+            value: {{ .Values.publisherProxy.resources.limits.memory }}
+          - name: PUBLISHER_IMAGE
+            value: "{{ .Values.global.image.repository }}/{{ .Values.publisherProxy.image.name }}:{{ .Values.publisherProxy.image.tag }}"
```

Comment:
The publisher "replicas" should be made configurable too, cause publisher could horizontally scale. It should be added into the "code" too.

Reply:
It will be solved by: https://github.com/kyma-project/kyma/issues/11348